clerk/app.py

```diff
@@ -107,24 +111,32 @@ def _apply_callbacks_for_hook(self, hook_name: str, filename: pathlib.Path):
     def open_journal(self, filename: str):
         """Opens the specified journal, calling appropriate Hooks along the way"""
         file_to_open: pathlib.Path = pathlib.Path(self.journal_directory, filename)
-        with tempfile.NamedTemporaryFile() as journal:
-            if not pathlib.Path(file_to_open).exists():
-                self._apply_callbacks_for_hook("NEW_JOURNAL_CREATED", journal.name)
-            else:
-                shutil.copy(file_to_open, journal.name)
-
-            self._apply_callbacks_for_hook("JOURNAL_OPENED", journal.name)
-
-            old_hash = get_file_hash(journal.name)
-            subprocess.run([self.preferred_editor, journal.name])
-            new_hash = get_file_hash(journal.name)
-
-            if old_hash != new_hash:
-                self._apply_callbacks_for_hook("JOURNAL_SAVED", journal.name)
-                shutil.copy(journal.name, file_to_open)
-
-            self._apply_callbacks_for_hook("JOURNAL_CLOSED", journal.name)
-            return True
+        temporary_copy: pathlib.Path = pathlib.Path(self.temp_directory, filename)
+        if temporary_copy.exists():
+            print("File already open!")
+            exit(1)
+        else:
+            f = open(temporary_copy, "a")
+            f.write("")
+            f.close()
```

Comment:
I added this block to fix failing tests when a new journal was created (_apply_callbacks_for_hook was complaining that no file existed) but I'm not sure this fix makes sense

should we instead check whether file_to_open exists, and create that instead?

but what if the user quits without saving? we wouldn't wanna create the file

Reply:
upon review, i think it's ugly but fine

previously we used `tempfile` to create the `temporary_copy`, but it was referenced via `journal.name` (since the [contextmanager created](https://github.com/josephhaaga/clerk/blob/ab7013104784f1796f23d997f9c066a8a704fbe9/clerk/app.py#L110-L112) `journal`)